app/src/main/java/io/left/meshim/activities/MainActivity.java

```diff
@@ -108,7 +110,14 @@ private void configureTabs() {
     }
 
     private View createTabIndicator(Context context, String title, int icon) {
-        View view = LayoutInflater.from(context).inflate(R.layout.tab_layout, null);
+        View view;
+        if (title.equals("Messages")) {
+            view = LayoutInflater.from(context).inflate(R.layout.tab_layout_messages, null);
```

Comment:
Is it possible to just have the badge in all tab layouts just left invisible?

If so, what you can do instead of modify this method is do
```
mMessageTabView = createTabIndicator(whatever);
spec.setIndicator(mMessageTabView);
```
And not have this logic depend on a string comparison.